apps/mull-ui/src/app/pages/create-event/location-autocomplete/location-autocomplete-textbox.tsx

```diff
@@ -3,46 +3,66 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import CircularProgress from '@material-ui/core/CircularProgress';
 import TextField from '@material-ui/core/TextField';
 import Autocomplete from '@material-ui/lab/Autocomplete';
+import { IGooglePlace } from '@mull/types';
+import axios from 'axios';
 import { debounce } from 'lodash';
-import React, { useEffect, useMemo, useState } from 'react';
-import { useAutocompletedLocationsLazyQuery } from '../../../../generated/graphql';
+import React, { useMemo, useState } from 'react';
+import { environment } from '../../../../environments/environment';
+import { LocationInput } from '../../../../generated/graphql';
+import { useToast } from '../../../hooks/useToast';
 import './location-autocomplete-textbox.scss';
 
 export interface LocationAutocompleteTextboxProps {
-  handleSetValue: (value: string) => void;
+  handleSetValue: (location: LocationInput) => void;
   input: string;
+  setInputValue: (value: React.SetStateAction<string>) => void;
 }
 
-export default function LocationAutocompleteTextbox({ handleSetValue, input }) {
+const CURRENT_LOCATION: IGooglePlace = { description: 'Current Location', placeId: null };
+
+export default function LocationAutocompleteTextbox({
+  handleSetValue,
+  input,
+  setInputValue,
+}: LocationAutocompleteTextboxProps) {
   const [open, setOpen] = useState(false);
-  const [options, setOptions] = useState<string[]>([]);
-  const [getAutocompletedLocations, { loading, data }] = useAutocompletedLocationsLazyQuery();
-  const CURRENT_LOCATION = 'Current Location';
+  const [options, setOptions] = useState<IGooglePlace[]>([]);
+  const [loading, setLoading] = useState<boolean>(false);
+  const { notifyToast } = useToast();
 
   const debounceGetLocation = useMemo(
     () =>
-      debounce(
-        (values: string) =>
-          getAutocompletedLocations({
-            variables: { userInput: values },
-          }),
-        350
-      ),
-    [getAutocompletedLocations]
+      debounce((search: string) => {
+        setLoading(true);
+        axios
+          .get(encodeURI(`${environment.backendUrl}/api/location-autocomplete?search=${search}`))
+          .then((data) => data.data)
+          .then((data: IGooglePlace[]) => {
+            setOptions([CURRENT_LOCATION, ...data]);
```

Comment:
This is so that users can still use the `Current Location` option after they started to type